nix/lib/gitea/mkGiteaStatus.nix

```diff
@@ -0,0 +1,69 @@
+utils: lib: {
+  mkGiteaStatus = {
+    instance,
+    owner,
+    repo,
+    typhon_url,
+  }:
+    lib.mkActionScript {
+      mkPath = system: let
+        pkgs = utils.pkgs.${system};
+      in [
+        pkgs.curl
+        pkgs.jq
+        pkgs.nix
+      ];
+      mkScript = system: ''
+        input=$(cat)
+
+        evaluation=$(echo "$input" | jq '.input.evaluation' -r)
+        job=$(echo "$input" | jq '.input.job' -r)
+        status=$(echo "$input" | jq '.input.status' -r)
+        system=$(echo "$input" | jq '.input.system' -r)
+        url=$(echo "$input" | jq '.input.url' -r)
+
+        token=$(echo "$input" | jq '.secrets.gitea_token' -r)
```

Comment:
Extracting one value at a time like that with `jq` is a bit repetitive, maybe we should have a small bash utility script we could share between the different actions that transform a JSON array into a bash one, wdyt?

Reply:
I want to factor out some code between `mkGithubProject` and `mkGiteaProject` anyway, so let's take the opportunity to refactor this in a following pr? Let's merge this first as it is already tested! (see https://codeberg.org/pnm/typhon-example)